examples/tv-casting-app/android/App/app/src/main/jni/com/matter/casting/support/TargetAppInfo.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2024 Project CHIP Authors All rights reserved.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.matter.casting.support;
+
+/**
+ * Feature: Target Content Application The set of content app Vendor IDs (and optionally, Product
+ * IDs) that can be used for authentication.
+ */
+public class TargetAppInfo {
+  public int vendorId = 0;
+  public int productId = 0;
```

Comment:
This is a case where in Java you can use the boxed type: Integer, assigned to null by default, to mean that no value has been set, instead of using a primitive int.

Reply:
We still need to initialize the Integer object with either "public Integer vendorId = 0;" or  "public Integer productId = new Integer(0);" 
Leaving it un-initialized will cause an JNI error in:
jobject jVendorIdObject  = env->GetObjectField(jTargetAppInfo, vendorIdField);
    jobject jProductIdObject = env->GetObjectField(jTargetAppInfo, productIdField);
    VerifyOrReturnValue(jVendorIdObject != nullptr, nullptr,
                        ChipLogError(AppServer, "convertTargetAppInfoFromJavaToCpp() vendorIdObject is null!"));
    VerifyOrReturnValue(jProductIdObject != nullptr, nullptr,
                        ChipLogError(AppServer, "convertTargetAppInfoFromJavaToCpp() productIdObject is null!"));